src/web/components/App.tsx

```diff
@@ -292,6 +295,18 @@ export const App = ({ CAPI, NAV }: Props) => {
         }
     }, [CAPI.shouldHideReaderRevenue]);
 
+    useEffect(() => {
+        if (CAPI.config.switches.consentManagement && countryCode) {
```

Comment:
Pass in CAPI.config.switches.consent directly?

Reply:
how do i do that? in the `useEffect` dependency list?

Reply:
Sorry, missed this comment. 

I meant, rather than re-run the effect on `CAPI`, in the array` pass in `CAPI.config.switches.consentManagement`? I think that's doable?

`}, [countryCode, CAPI.config.switches.consentManagement]);`